src/OrleansTelemetryConsumers.Counters/OrleansPerfCounterTelemetryConsumer.cs

```diff
@@ -187,8 +189,7 @@ private bool Initialize()
                 var newPerfCounterData = new List<PerfCounterConfigData>(PerfCounterConfigData.StaticPerfCounters);
 
                 // TODO: get rid of this static access. Telemetry consumers now allow being injected with dependencies, so extract it as such
-                var grainTypes = LogManager.GrainTypes;
-                if (grainTypes != null)
+				var grainTypes = CrashUtils.GrainTypes;                if (grainTypes != null)
```

Comment:
hmmm, CrashUtils? I know @ReubenBond is going to avoid this hack in the future, but crash utils? BTW, linefeed broken

Reply:
CrashUtils is named that because it contains CreateMiniDump method. CrashUtils is not kept there because @ReubenBond 's hack. @ReubenBond can remove his hack from it.